django/cantusdb_project/main_app/urls.py

```diff
@@ -72,7 +72,11 @@
     DifferentiaAutocomplete,
     ProvenanceAutocomplete,
     ProofreadByAutocomplete,
+    handle400,
```

Comment:
I don't think you need these imports since you're not using these directly. You could change your code below to not express these as strings, but the way it is now they're unused.